arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts

```diff
@@ -6,51 +6,109 @@
 /dts-v1/;
 /plugin/;
 
+#include "edt-ft5406.dtsi"
+
 / {
 	compatible = "brcm,bcm2835";
 
 	fragment@0 {
 		target = <&dsi1>;
 		__overlay__ {
-			#address-cells = <1>; size-cells = <0>;
+			#address-cells = <1>;
+			#size-cells = <0>;
 			status = "okay";
 			port {
-				dsi_out_port: endpoint {
-					remote-endpoint = <&panel_dsi_port>;
+				dsi_out: endpoint {
+					remote-endpoint = <&bridge_in>;
+				};
+			};
+			bridge@0 {
+				reg = <0>;
+				compatible = "toshiba,tc358762";
+				vddc-supply = <&reg_bridge>;
+				ports {
+					#address-cells = <1>;
+					#size-cells = <0>;
+
+					port@0 {
+						reg = <0>;
+						bridge_in: endpoint {
+							remote-endpoint = <&dsi_out>;
+						};
+					};
+
+					port@1 {
+						reg = <1>;
+						bridge_out: endpoint {
+							remote-endpoint = <&panel_in>;
+						};
+					};
 				};
 			};
 		};
 	};
 
 	fragment@1 {
+		target-path = "/";
+		__overlay__ {
+			panel_disp1: panel_disp1@0 {
```

Comment:
What do the addresses + `reg`s on these nodes achieve?

Reply:
We have 2 DSI interfaces, so 2 instances of this overlay is possible with i2c_csi_dsi -> i2c_vc, and dsi1 -> dsi0. Regulators, panels, bridges, etc therefore all need to be unique.
Whilst we haven't done the override magic as yet, it seemed to make sense to be prepared.

Reply:
Understood.